PRIVACY.md

```diff
@@ -130,45 +149,396 @@ provided by [http://www.aboutads.info/choices/](http://www.youronlinechoices.com
 [http://www.youronlinechoices.com/uk/your-ad-choices/](http://www.youronlinechoices.com/uk/your-ad-choices) for users form the
 European Union.
 
-# 9. Use of Twitter Buttons
-
-9.1. This website uses Twitter-Buttons (e.g. "Twitter this", "Twitter", "Follow
-@twitter") provided by the service Twitter, operated by Twitter Inc., 795
-Folsom St., Suite 600, San Francisco, CA 94107, USA. When a user visits a page
-of this website that contains a Twitter-Button, his browser establishes a
-direct connection to Twitter servers. Twitter directly transfers the
-Twitter-Button's content to user's browser which embeds the latter into the
-website, enabling Twitter to receive information about the user having accessed
-the respective page of this website. Thus the Service Provider have no
-influence on the data gathered by Twitter and informs users according to its
-state of knowledge: In accordance to Twitter, its buttons doesn't collect any
-personally identifiable data on users. If you are a Twitter member and do not
-want Twitter to connect the data concerning your visit to our website with your
-member data already stored by Twitter, please log off from Twitter and delete
-all Twitter related cookies in your browser before entering our website.
-
-9.2. For the information on the purpose and scope of data collection and
-procession by Twitter, as well as user's rights in this respect and settings
-options for protecting user's privacy please visit Twitter's privacy policy:
-[http://twitter.com/privacy](http://twitter.com/privacy).
-
-# 10. Contact and Notices
-
-10.1. In compliance with legal requirements we will at any time provide you
-with information on your personally identifiable data stored by us, free of
-charge and without undue delay.
+# 9. Payment Transaction
```

Comment:
§9 - §14 describe all the third party subcontractors we use to fulfill our service and contract with you in detail and what we use them for.